common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/BlockStoreClient.java

```diff
@@ -61,4 +78,56 @@ public MetricSet shuffleMetrics() {
     // Return an empty MetricSet by default.
     return () -> Collections.emptyMap();
   }
+
+  /**
+   * Request the local disk directories for executors which are located at the same host with
+   * the current BlockStoreClient(it can be ExternalBlockStoreClient or NettyBlockTransferService).
+   *
+   * @param host the host of BlockManager or ExternalShuffleService. It's the same with current
+   *             BlockStoreClient.
```

Comment:
What happens when we break this assumption, `It's the same with current BlockStoreClient`? I believe we can remove this comment if this is a general reusable function for all remote host.

Reply:
It's a pre-condition which should be guaranteed by the caller. I've changed it to `should be`. Thanks!